backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/schema/create/Organization.kt

```diff
@@ -0,0 +1,27 @@
+package app.ehrenamtskarte.backend.application.webservice.schema.create
+
+import app.ehrenamtskarte.backend.application.webservice.schema.view.JsonField
+import app.ehrenamtskarte.backend.application.webservice.schema.view.Type
+import app.ehrenamtskarte.backend.application.webservice.utils.JsonFieldSerializable
+import com.expediagroup.graphql.annotations.GraphQLDescription
+
+data class Organization(
+    val name: String,
+    val address: String,
```

Comment:
```suggestion
    val address: String,
    val category: String,
```

Reply:
@PaulaWi What is the meaning of category and responsibility?

Reply:
Category is something like Sport, Umweltschutz, Soziales, Feuerwehr and responsibility is a description of the kind of work the volunteer does.

Reply:
Should Category not be `categories`, i.e. do we allow multiple categories? Or is this a free-text field?